catalyst-gateway/bin/src/service/common/types/cardano/catalyst_id.rs

```diff
@@ -35,15 +41,12 @@ impl Type for CatalystId {
     const IS_REQUIRED: bool = true;
 
     fn name() -> Cow<'static, str> {
-        "string(catalyst_id)".into()
+        format!("string{FORMAT}").into()
```

Comment:
```suggestion
        format!("string({FORMAT})").into()
```
Should there be braces?